src/main/java/com/flightstats/hub/model/BulkContent.java

```diff
@@ -11,16 +13,29 @@
     private final InputStream stream;
     private final String contentType;
     private final String channel;
-    private final List<Content> items = new ArrayList<>();
+    private final List<Content> items;
     private ContentKey masterKey;
 
-    @java.beans.ConstructorProperties({"isNew", "stream", "contentType", "channel", "masterKey"})
-    BulkContent(boolean isNew, InputStream stream, String contentType, String channel, ContentKey masterKey) {
+    @java.beans.ConstructorProperties({"isNew", "stream", "contentType", "channel", "masterKey", "items"})
+    BulkContent(boolean isNew, InputStream stream, String contentType, String channel, ContentKey masterKey, List<Content> items) {
         this.isNew = isNew;
         this.stream = stream;
         this.contentType = contentType;
         this.channel = channel;
         this.masterKey = masterKey;
+
+        if (items == null) {
+            this.items = new ArrayList<>();
+        } else {
+            this.items = items;
+        }
+    }
+
+    public static BulkContent fromMap(String channelName, Map<ContentKey, Content> map) {
+        return BulkContent.builder()
+                .channel(channelName)
+                .items(map.values().stream().collect(Collectors.toList()))
```

Comment:
IntelliJ wants this to be simplified.

Reply:
IntelliJ doesn't have anything to say for me with regard to that line. What does it want it to be?